assets/helpers/utilities.js

```diff
@@ -42,3 +42,16 @@ export function clickSubstituteKeyPressHandler(handler?: () => void = () => {})
   };
 }
 
+// Attempts to parse a boolean from a string.
+export function parseBoolean(boolString: ?string): ?boolean {
```

Comment:
Why is `string` optional?

Reply:
Because `getQueryParameter` returns an optional string.

Reply:
That doesn't seem like it is any of `utilities.parseBoolean`'s concern.  At the very least the comment should change to "Attempts to parse a boolean from an optional string"

Reply:
Updated.